language/stdlib/modules/Coin1.move

```diff
@@ -24,6 +25,39 @@ module Coin1 {
         Libra::publish_mint_capability<Coin1>(tc_account, coin1_mint_cap, tc_account);
         Libra::publish_burn_capability<Coin1>(tc_account, coin1_burn_cap, tc_account);
     }
+
+// **************** SPECIFICATIONS  ****************
+
+/// Only TreasuryComplianceRole may mint Coin1
+
+/// BUG: This property verifies, but it does not necessarily hold.
+/// Libra.move has a function `remove_mint_capability<CoinType>` that removes
```

Comment:
I think there's no problem with getting rid of `remove_mint_capability` now that we don't use it in genesis anymore. We may need it later if treasury compliance needs the flexibility to mint the same coin from multiple accounts, but that is hypothetical. I would vote to eliminate!

CC @tzakian

Reply:
Ditto! Happy to see this function go. If we need to add it back in at a later date it's a code update, so not the worst thing in the world.